### DISCUSSION TITLE
What are the required ports to portforward Agent DVR?

### DISCUSSION THREAD

**Discussion by kgomez6** [doitandbedone/ispyagentdvr-docker#723] [Q&A]
I have purchase the business full licence, and I wanted to be able to access the agent dvr outside my network through portforward, but I am not sure what port should I open since whenever I tried accessing the public ip with the port it says "Connecting" forever.

**Comment by GentlemanMoose** [comment]
I am using a website built on nodejs to access my agentdvr, I needed to open up 80, 443 and 3478 on my router to finally get it working. Also make sure they are open on the firewall on the host computer to the container. 3478 is also UDP only.

**Comment by ispysoftware** [comment]
see

https://www.ispyconnect.com/docs/agent/remoteaccess#port-forward-remote-access
